frontend/src/metabase-lib/v1/Question.ts

```diff
@@ -105,7 +105,7 @@ class Question {
   constructor(
     card: any,
     metadata?: Metadata,
-    parameterValues?: ParameterValues,
+    parameterValues?: ParameterValues | Record<ParameterId, Parameter["value"]>,
```

Comment:
When can it be `ParameterValues` ?

```suggestion
    parameterValues?: Record<ParameterId, Parameter["value"]>,
```

Reply:
Please also update the type in line 96 (`_parameterValues: ParameterValues`).